cbmc/.gitignore

```diff
@@ -0,0 +1,6 @@
+proofs/ninja.py
```

Comment:
Is it good etiquette to add to the .gitignore something that is our own development tool?  This is actually kind of dangerous, because I routinely do "git clean -fdx" to blow away all my trash, and this will remove the development tools listed in gitignore.

Reply:
Done!